src/sql/src/kafka_util.rs

```diff
@@ -83,6 +89,34 @@ generate_extracted_config!(
     (RetentionMs, i64)
 );
 
+impl TryFromValue<Value> for KafkaSinkCompressionType {
+    fn try_from_value(v: Value) -> Result<Self, PlanError> {
+        match v {
+            Value::String(v) => match v.as_str() {
```

Comment:
do we care about case sensitivity on input?

Reply:
Oh wow. I thought librdkafka was case sensitive here but it absolutely wasn't. Good catch. Fixed.

Reply:
And added a test too.